### PR TITLE
First version from eos ntuples to plots

### PR DESCRIPTION
**Pull request by clementhelsens**
The documentation is still missing, but this first version demonstrate how we can run with RDF from FCCSW ntuples on eos to doing plots. Everything is in the same 

Each analysis is hosted in a single directory, for example **FCCeeAnalyses/ZH_Zmumu/dataframe/** and contains the same kind of files:

- `analysis.py`: this is a class that is used to define the list of analysers/filters to run and the output variables.
- `preSel.py`: used to define the list of samples, number of CPUs, fraction of the original sample to run on and base directory for the yaml. This will run the `analysis.py` with a common code `bin/runDataFrame.py`, in this example`python FCCeeAnalyses/ZH_Zmumu/dataframe/preSel.py`
- `finalSel.py`: This is where the final selections are implemented. It runs over the files produced at the pre-selection level. It contains the procDict for cross section etc..., list of processes, number of CPU, the cut list, and the variables (that will be both written in a TTree and in the form of TH1 properly normalised to an integrated luminosity of 1pb-1. In this example it should be run like: `python FCCeeAnalyses/ZH_Zmumu/dataframe/finalSel.py` which calls a common code `bin/runDataFrameFinal.py` to run dataframe.
- `plots.py`: This is where the configuration of the selections to plot is done, with how merging processes, etc... For the moment it is possible to only plot one signal at the time, but several backgrounds. In this example just run like: `python bin/doPlots.py FCCeeAnalyses/ZH_Zmumu/dataframe/plots.py`

### CODE REVIEW

File: analyzers/dataframe/FCCAnalyses.cc [hunk 1]
```diff
@@ -220,6 +220,16 @@ int get_njets2(ROOT::VecOps::RVec<fcc::JetData> x, ROOT::VecOps::RVec<fcc::JetDa
   return result;
 }
 
+float get_flat(ROOT::VecOps::RVec<float> x, int i)
```
Comment (vvolkl) [inline analyzers/dataframe/FCCAnalyses.cc]: I think this doesn't work, no? Could be removed

**Comment by vvolkl** [comment]
Great! One question, does the plotting code do everything the FlatTreeAnalyzer does?

File: analyzers/dataframe/FCCAnalyses.cc (hunk 1 above)
Comment (clementhelsens) [reply analyzers/dataframe/FCCAnalyses.cc]: yes, true, this does not work and from to dump into histogram this is not needed. will remove

**Comment by clementhelsens** [comment]
Yes @vvolkl , the plotting does even more than the FlatTreeAnalyzer code. I improved the logic to make it more flexible.

**Pull request merged**